src/main/java/core/basesyntax/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.User;
+import core.basesyntax.storage.ListStorageUser;
+
+public class UserDaoImpl implements UserDao {
+    @Override
+    public void addNewUser(User user) {
+        ListStorageUser listStorageUser = new ListStorageUser();
+        listStorageUser.addUser(user);
```

Comment:
Dao should work with database. And in your database you have:
public static List userList
It is static and you can call it by the class name: ListStorageUser.userList.add(user)
So it is no need to create new ListStorageUser.

Reply:
ok